api.js

```diff
@@ -132,11 +132,8 @@ Api.prototype.observationUpdate = function (req, res, m) {
         })
         opts.links = old.id
       }
-      var newObs = Object.assign(old, {
-        properties: obs.properties,
```

Comment:
`m` is the match of parameters, not the observation. `obs` is the new observation from the client.

Reply:
ok fixed